software/reposilite/README.md

```diff
@@ -0,0 +1,11 @@
+# [Reposilite](https://github.com/dzikoysk/reposilite)
+
+Lightweight and easy-to-use repository management software dedicated for Maven based artifacts in the JVM ecosystem.
+
```

Comment:
Pterodactyl doesn't allow or can use privileged ports below 1024, thus displaying 80 is misleading that might confuse some users.

Reply:
I'm not sure what to put there then, since the Reposilite default is 80, but the egg adjusts the port to the `{{SERVER_PORT}}` variable

Reply:
You can set it to `Any` as an example